scalameter-core/src/main/scala/org/scalameter/keys.scala

```diff
@@ -16,11 +18,25 @@ class Key[T](val name: String)(implicit container: KeyContainer) extends Seriali
     case k: Key[_] => name == k.name
     case _ => false
   }
+
+  @annotation.tailrec
```

Comment:
Please import `scala.annotation.tailrec`.

If this method is only used by `key`, make it a nested method in the `key` initializer block.